### ISSUE TITLE
Create a Table Admin Sample

### ISSUE THREAD

**Issue by sduskis** [googleapis/nodejs-bigtable#116]
There needs to be a sample in the samples/ directory that manipulates tables. Here's the list of commands:

- **createTable [projectId] [instanceId]**
    - Creates a table named "my_table" with 1 column family "simpleFamily" that has a max versions of 1
- **checkTableExistence  [projectId] [instanceId]**
    - Does a list tables, and displays true if the table is in the list.
- **getTable [projectId] [instanceId]**
    - Gets "my_table", and displays the column families and their gcPolicy
- **listTables  [projectId] [instanceId]**
    - // lists all of the tables
- **deleteTable [projectId] [instanceId]**
   - Deletes "my_table"
- **addComplexColumnFamily [projectId] [instanceId]**
   - Creates a column family called "complex" on "my_table" that has the following GCRule which means "have at least 2 columns, but no more than 10.  Any additional columns past 2 should have a TTL of 30d: 
```
   union: [
       {max_versions: 10}
       {intersection: [
         {max_versions: 2}
         {max_age: 30d}
       ]}
   ]
```
- **deleteComplexColumnFamily [projectId] [instanceId]**
   - Deletes the "complex" column family

**Comment by jabubake** [comment]
@vijay-qlogic @stephenplusplus (Sorry for the overlap in #139), We are moving to supporting minimal CLI commands (for these samples, since some of them are best showcased in sequence)
https://github.com/googleapis/nodejs-bigtable/pull/139/files#diff-09aca9530f1bfc5bd57e7988fc393745.

I'm happy to pull in your code to list tables in a project and the complex column family, so that this PR can be closed. 

@sduskis FYI

**Comment by sduskis** [comment]
This was fixed by #139
